src/imports/Components/Pickers/1.2/Picker.qml

```diff
@@ -221,7 +221,7 @@ StyledItem {
             target: loader.item
             property: "currentIndex"
             value: picker.selectedIndex
-            when: loader.completed && (picker.selectedIndex > 0)
+            when: picker.selectedIndex > 0
```

Comment:
It creates a binding loop as the `Binding` component here is a child of the `Loader` and it depends on `loader.item` which is undefined until the loader has completed loading it, IIRC.

Reply:
Hmm, I'm not sure why it caused a binding loop. `Binding` should be able to handle `undefined` `target` just fine. Apparently `completed` is a bit more than the `Loader` having loaded the component.

Reply:
With `loader.completed` there, it results in the following warning:
```
file:///build/ubuntu-ui-toolkit-1.3.2192~0ubports2/qml/Ubuntu/Components/Pickers/1.2/Picker.qml:220:9: QML Binding: Binding loop detected for property "when"
```
I'm pretty sure this is because the `Binding` is a direct child of `loader` here.